front/lib/api/assistant/global_agents.ts

```diff
@@ -858,6 +859,69 @@ async function _getDustGlobalAgent(
     };
   }
 
+  const actions: AgentActionConfigurationType[] = [
```

Comment:
Interesting ! Do we want to update the instructions a bit to clarify that "all" should be used unless it's very clear from conversation history that the search should focus a specific DS ?

Reply:
Good idea, I added a commit!